content/docs/reference/glossary.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Pulumi Glossary
```

Comment:
Wondering if we could get rid of `Pulumi` and just have it be Glossary?

Reply:
I took your suggestion.